src/background/listener/bee-api.listener.ts

```diff
@@ -276,8 +276,25 @@ export class BeeApiListener {
    * @param bzzReference in form of $ROOT_HASH<$PATH><$QUERY>
    * @param tabId the tab will be navigated to the dApp page
    */
-  private redirectToBzzReference(bzzReference: string, tabId: number) {
-    const url = `${this._beeApiUrl}/bzz/${bzzReference}`
+  private redirectToBzzReference(bzzReference: string, tabId: number, preventSubdomainRedirection = false) {
```

Comment:
remove `preventSubdomainRedirection
```suggestion
  private redirectToBzzReference(bzzReference: string, tabId: number) {
```